keyboards/mechlovin/adelais/rgb_led/rev2/rules.mk

```diff
@@ -0,0 +1,2 @@
+RGB_MATRIX_ENABLE = IS31FL3731
+RGBLIGHT_ENABLE = yes
```

Comment:
You may want to define `RGB_MATRIX_DISABLE_KEYCODES` or `RGBLIGHT_DISABLE_KEYCODES` in the config.h for this revision, so that the rgb keycodes don't do both sets of functions.

Reply:
oh, its new for me. 
Thanks.

Reply:
I had a PR for it for a while, and it was merged recently. So ... :)